pom.xml

```diff
@@ -1,334 +1,254 @@
-    <artifactId>mp-istio-sample-servicea</artifactId>
-    <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+  <groupId>projects</groupId>
+  <artifactId>service-a</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
 
-    <!-- contains default pluginManagement configuration for liberty-maven-plugin -->
-    <parent>
-        <groupId>net.wasdev.wlp.maven.parent</groupId>
-        <artifactId>liberty-maven-app-parent</artifactId>
-        <version>2.2</version>
-    </parent>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
 
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+    <app.name>service-a</app.name>
+    <testServerHttpPort>9080</testServerHttpPort>
+    <testServerHttpsPort>9443</testServerHttpsPort>
+    <warContext>${app.name}</warContext>
+    <package.file>${project.build.directory}/${project.artifactId}-${project.version}.zip</package.file>
+    <packaging.type>usr</packaging.type>
 
-        <app.name>mp-istio-sample-a</app.name>
-        <testServerHttpPort>9080</testServerHttpPort>
-        <testServerHttpsPort>9443</testServerHttpsPort>
-        <warContext>${app.name}</warContext>
-        <package.file>${project.build.directory}/${project.artifactId}-${project.version}.zip</package.file>
-        <packaging.type>usr</packaging.type>
+    <!-- Plugins -->
+    <version.maven-war-plugin>2.6</version.maven-war-plugin>
+    <version.liberty-maven-plugin>2.2</version.liberty-maven-plugin>
+    <version.download-maven-plugin>1.4.0</version.download-maven-plugin>
+    <version.maven-surefire-plugin>2.18.1</version.maven-surefire-plugin>
+    <version.maven-failsafe-plugin>2.18.1</version.maven-failsafe-plugin>
 
+    <!-- OpenLiberty runtime -->
+    <version.openliberty-runtime>RELEASE</version.openliberty-runtime>
+    <version.exec-maven-plugin>1.6.0</version.exec-maven-plugin>
+  </properties>
 
-        <!-- Plugins -->
-        <version.maven-war-plugin>2.6</version.maven-war-plugin>
-        <version.liberty-maven-plugin>2.2</version.liberty-maven-plugin>
-        <version.download-maven-plugin>1.4.0</version.download-maven-plugin>
-        <version.maven-surefire-plugin>2.18.1</version.maven-surefire-plugin>
-        <version.maven-failsafe-plugin>2.18.1</version.maven-failsafe-plugin>
-        <!-- OpenLiberty runtime -->
-        <version.openliberty-runtime>RELEASE</version.openliberty-runtime>
-        <version.exec-maven-plugin>1.6.0</version.exec-maven-plugin>
+  <dependencies>
```

Comment:
Any dependencies that are specific to Liberty, such as CXF, should only be part of the Liberty profile